Successfully phing target product-search-create-structure with error

### What is happening
Phing target `product-search-create-structure` is successfully finished without exists JSON configuration for new domain.

Add new domain, run `build-demo-dev`, fix datafixtures...
Build fails on this error:
```
Shopsys Framework > product-search-export-products:

Exporting products to microservice
05:32:49 ERROR     [app] Elasticsearch\Common\Exceptions\Missing404Exception: {"error":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index","resource.type":"index_or_alias","resource.id":"product3","index_uuid":"_na_","index":"product3"}],"type":"index_not_found_exception","reason":"no such index","resource.type":"index_or_alias","resource.id":"product3","index_uuid":"_na_","index":"product3"},"status":404} (uncaught exception) at /var/www/html/vendor/elasticsearch/elasticsearch/src/Elasticsearch/Connections/Connection.php line 607 while running console command `shopsys:product-search:export-products` ["exception" => Elasticsearch\Common\Exceptions\Missing404Exception { …}]
```
It is very weird for me... Now I know, that I forgot to create `3.json` but firstly I was looking for a mistake elsewhere. Later I noticed error during phing target `product-search-create-structure` like this:
```
[ERROR] File /var/www/html/app/../src/Shopsys/ShopBundle/Resources/definition//product/3.json not found
```
From this error I dont have idea what was happend... What is it `3.json` in this general filepath? Ok, maybe error is double slash in this filepath... Oh, wait! 3.json is probably something for new domain.

### Expected result
I expect failing `product-search-create-structure` target without all needed configs.
I want better descriptive error on missing this config files
I hope for better filepath or filename for this configs (I am going to create next issue for move this configurations from JSON to PHP)

--- a/packages/framework/src/Command/ProductSearchCreateStructureCommand.php
+++ b/packages/framework/src/Command/ProductSearchCreateStructureCommand.php
@@ -2,7 +2,6 @@
 
 namespace Shopsys\FrameworkBundle\Command;
 
-use Shopsys\FrameworkBundle\Component\Elasticsearch\Exception\ElasticsearchStructureException;
 use Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportStructureFacade;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
@@ -44,11 +43,7 @@ class ProductSearchCreateStructureCommand extends Command
     {
         $symfonyStyleIo = new SymfonyStyle($input, $output);
         $output->writeln('Creating structure');
-        try {
-            $this->productSearchExportStructureFacade->createIndexes($output);
-            $symfonyStyleIo->success('Structure created successfully!');
-        } catch (ElasticsearchStructureException $e) {
-            $symfonyStyleIo->error($e->getMessage());
-        }
+        $this->productSearchExportStructureFacade->createIndexes($output);
+        $symfonyStyleIo->success('Structure created successfully!');
     }
 }

--- a/packages/framework/src/Component/Elasticsearch/ElasticsearchStructureManager.php
+++ b/packages/framework/src/Component/Elasticsearch/ElasticsearchStructureManager.php
@@ -85,7 +85,13 @@ class ElasticsearchStructureManager
     {
         $file = sprintf('%s/%s/%s.json', $this->definitionDirectory, $index, $domainId);
         if (!is_file($file)) {
-            throw new ElasticsearchStructureException(sprintf('File %s not found', $file));
+            throw new ElasticsearchStructureException(
+                sprintf(
+                    'Definition file %d.json, for domain ID %1$d, not found in definition folder "%s".' . PHP_EOL . 'Please make sure that for each domain exists a definition json file named by the corresponding domain ID.',
+                    $domainId,
+                    $this->definitionDirectory
+                )
+            );
         }
         $json = file_get_contents($file);
 

